src/test/java/com/commercetools/service/paypalPlus/constant/PaypalPlusPaymentStates.java

```diff
@@ -0,0 +1,10 @@
+package com.commercetools.service.paypalPlus.constant;
+
+public final class PaypalPlusPaymentStates {
```

Comment:
maybe in future, but so far these values are treated exclusively as strings, so i don't want to stick with permanent converting `Enum#toString()`, `Enum#valueOf()` with catching _IllegalArgumentException_